src/app/components/views/home/home.component.scss

```diff
@@ -0,0 +1,88 @@
+
+.designer {
+    position: absolute;
+    left: 13%;
```

Comment:
Try to leverage bootstrap columns instead of these.

Reply:
Refer [this](https://jsfiddle.net/isherwood/an38qf0y/1/) for text overlay

Reply:
for what all screen sizes do i need to do?

Reply:
If you use bootstrap you don't need to write separate styles for every screen size. You can specify how you want for large, small, medium screens as classes. See [this](https://getbootstrap.com/docs/4.0/layout/grid/)

Reply:
done